app/controllers/doctors/consultation_request_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Doctors::ConsultationRequestController < ApplicationController
+  layout 'doctors/doctors'
```

Comment:
If you inherit from the DoctorController, you can remove line 2 - as it's already defined in the DoctorController 👍